src/components/bulkConcepts/bulkConceptList.jsx

```diff
@@ -1,29 +1,66 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import BulkConceptTable from './bulkConceptTable';
+import ReactTable from 'react-table';
 import Loader from '../Loader';
 
-const BulkConceptList = ({ cielConcepts, fetching, handleSelect }) => {
+const BulkConceptList = ({
+  cielConcepts,
+  fetching,
+  handleSelect,
+  conceptLimit,
+  filterConcept,
+}) => {
+  const filter = { filterMethod: filterConcept, filterAll: true };
   if (cielConcepts.length >= 1) {
     return (
-      <table className="table table-bordered table-striped">
-        <thead className="header text-white">
-          <tr>
-            <th scope="col">Select</th>
-            <th scope="col">Name</th>
-            <th scope="col">Datatype</th>
-            <th scope="col">Source</th>
-            <th scope="col">ID</th>
-          </tr>
-
-        </thead>
-        <tbody>
-
-          {cielConcepts.map(concept => (
-            <BulkConceptTable concept={concept} key={concept.id} handleSelect={handleSelect} />
-          ))}
-        </tbody>
-      </table>
+      <ReactTable
+        data={cielConcepts}
+        loading={fetching}
+        defaultPageSize={cielConcepts.length <= conceptLimit ? cielConcepts.length : conceptLimit}
+        filterable
+        noDataText="No ceoncept!"
```

Comment:
@dessHub, I think you meant `No concept!` here. Please take a look at the spelling.

Reply:
Ooh yeah, on it

Reply:
Fixed it.